### PR TITLE
Add CTSI

### PR DESCRIPTION
**Pull request by colinsteil**
**Description**
Add CTSI to the Optimism Bridge.

### CODE REVIEW

**Comment by colinsteil** [comment]
Hey. Is there anything required on my end in order for this to pass? Much appreciated!

**Comment by colinsteil** [comment]
Hey, sorry, is there anything I need to do in order for it to validate? Much appreciated!

**Comment by smartcontracts** [comment]
Hey @colinsteil sorry about the delayed response here, we had some issues with the way we were leaving automated comments. I'm trying to get the bot running again. In the meantime, here's the error that was generated:

```
undefined:2
  "name": “Cartesi Token”,
          ^

SyntaxError: Unexpected token “ in JSON at position 12
    at JSON.parse (<anonymous>)
    at /home/runner/work/ethereum-optimism.github.io/ethereum-optimism.github.io/src/generate.js:27:25
    at Array.map (<anonymous>)
    at generate (/home/runner/work/ethereum-optimism.github.io/ethereum-optimism.github.io/src/generate.js:26:6)
    at validate (/home/runner/work/ethereum-optimism.github.io/ethereum-optimism.github.io/src/validate.js:247:16)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Command.<anonymous> (/home/runner/work/ethereum-optimism.github.io/ethereum-optimism.github.io/bin/cli.js:21:21)
```

This is probably because you're using “ instead of " (smart quotes instead of normal quotes).

**Comment by colinsteil** [comment]
Hey @smartcontracts got it, thanks a bunch! I've just updated these - let me know if all good. Cheers

File: data/CTSI/data.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Cartesi Token",
+  "symbol": "CTSI",
+  "decimals": 18,
+  "description": "Cartesi - The Blockchain OS is a decentralized Layer-2 infrastructure that supports Linux and mainstream software components. For the first time, developers can code scalable smart contracts with rich software tools, libraries, and the services they’re used to, bridging the gap between mainstream software and blockchain.
```
Comment (smartcontracts) [inline data/CTSI/data.json:5]: Last thing here -- make sure not to use newlines in the description, instead use `\n` where you want newlines to appear. But probably better to just not use newlines at all.

**Comment by colinsteil** [comment]
Hey @smartcontracts all good? Thanks for the help!

File: data/CTSI/data.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "tokens": {
+    "ethereum": {
+      "address": "0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D"
+    },
+    "optimism": {
+      "address": "0xEc6adef5E1006bb305bB1975333e8fc4071295bf"
+    },
+    "optimism-kovan": {
+      "address": "0xEc6adef5E1006bb305bB1975333e8fc4071295bf"
+    },
```
Comment (roninjin10) [inline data/CTSI/data.json:17]: one last issue this comma needs to be removed for this to be valid jsonone last issue this comma needs to be removed for this to be valid json

**Review by roninjin10 — CHANGES_REQUESTED** [review]

**Pull request merged**